contracts/interfaces/external/IERC4626.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.0) (interfaces/IERC4626.sol)
+
+pragma solidity 0.6.10;
```

Comment:
@ckoopmann can't remember the consensus on this - should all new contracts be on the 0.8 solidity compiler?

Reply:
Yes,  those that don't have any  dependencies (inheritance) on old 0.6 contracts like this one should be using 0.8 solidity imo.

Reply:
Yes, imo contracts like this one that dont have a bunch of inheritance dependencies on old 0.6 set contracts should just be written directly in 0.8.

Reply:
upgraded to solidity `0.8.17` here [7a0aa20](https://github.com/IndexCoop/index-protocol/pull/11/commits/7a0aa20cb5036ecefb25b4fe2aa1e7c166ebd07b)